### PR TITLE
Comments: design tweaks

### PR DESCRIPTION
**Pull request by drw158**
- Make pending label less buttony
- Tweaks to reply field

Before

screen shot 2017-06-29 at 11 34 45 am

After

screen shot 2017-06-28 at 4 07 43 pm

### CODE REVIEW

File: client/blocks/comment-detail/comment-detail-reply.jsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ export class CommentDetailReply extends Component {
 							onChange={ this.handleTextChange }
 							onFocus={ this.setFocus }
 							onKeyDown={ this.submitCommentOnCtrlEnter }
-							placeholder={ translate( 'Enter your comment here…' ) }
+							placeholder={ translate( 'Reply to comment…' ) }
```
Comment (a8ci18n) [inline client/blocks/comment-detail/comment-detail-reply.jsx]: Hi! I've found a possible matching string that has already been translated **95** times:
`translate( 'Reply to Comment' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments-tweaks)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by Copons** [comment]
@drw158 Thanks for the changes, but I'll have to ask you to put this PR on hold.

The send button position appears incorrect because the vertical scrollbar is missing.
When I positioned it so far from the right edge it's because I was taking into consideration the scrollbar width, which I somehow left out from the previous PR.

Also, in the work I'm doing to enable the resize, I'm fiddling around with the entire textarea box model, borders included.

It's taking me longer than expected, but I hope to open a PR tomorrow.

**Comment by drw158** [comment]
Updated branch. @Copons I could use another review, thanks!

**Comment by gwwar** [comment]
@drw158 did you need help rebasing?

**Comment by drw158** [comment]
@gwwar Does it need one now? Looks like I did one at some point... I remember now having some trouble, but it looks good to me currently.

**Comment by gwwar** [comment]
There was a master merge, and you still have conflicts 😄

**Comment by drw158** [comment]
Oh jeez, maybe it was because I used GitHub.com to resolve the conflicts. Maybe I do need help with the rebase... right now I don't see any conflicts though.

**Comment by gwwar** [comment]
@drw158 👍 I found out that this is only complaining about the rebase and merge option. I think we'll be ok here but please be sure to choose the squash and merge later, since it's best to avoid a master merge cluttering history.

screen shot 2017-07-11 at 12 28 15 pm

> I used GitHub.com to resolve the conflicts.

😄 perhaps do that locally in the future?

**Comment by drw158** [comment]
Will try that next time. So ready to 🚢 with the squash and merge option?

**Pull request merged**